tests/test_pe_source.py

```diff
@@ -335,5 +336,26 @@ def test_shodan_search(
         )
 
 
+def test_dnstwistfuzzing():
+    """Test if dnstwist is installed correctly."""
+    assert pe_source.dnstwistscript.execute_dnstwist("a.com", test=1) != []
+
+
+def test_blocklist():
+    """Test if blocklist is working correctly."""
+    dom = {
+        "fuzzer": "addition",
+        "domain": "a0.com",
+        "dns_ns": ["liz.ns.cloudflare.com"],
+        "dns_a": ["104.21.34.160"],
+        "dns_aaaa": ["2606:4700:3036::6815:22a0"],
+        "dns_mx": ["alt1.aspmx.l.google.com"],
+        "ssdeep_score": "",
+    }
+    test1, test2 = pe_source.dnstwistscript.checkBlocklist(dom, 1, 1, 1, [])
+    assert len(test1) == 16
+    assert len(test2) == 1
```

Comment:
Can you please explain how the length of these outputs confirms that the blocklist is working correctly?

Reply:
replaces to check if certainvalues within the result were correct

Reply:
commit f13aeb9b862c936b00584c941a5efcaad5df7afc

Reply:
Much better, thanks!